js/apps/admin-ui/src/realm-settings/user-profile/attribute/AddTranslationsDialog.tsx

```diff
@@ -234,48 +255,53 @@ export const AddTranslationsDialog = ({
                         </Tr>
                       </Thead>
                       <Tbody>
-                        {filteredLocales.map((locale, rowIndex) => (
-                          <Tr key={rowIndex}>
-                            <Td
-                              className="pf-m-sm pf-u-px-sm"
-                              dataLabel={t("supportedLanguage")}
-                            >
-                              <FormGroup fieldId="kc-supportedLanguage">
-                                {localeToDisplayName(
-                                  locale,
-                                  whoAmI.getLocale(),
-                                )}
+                        {filteredLocales.map((locale) => {
+                          const rowIndex = combinedLocales.findIndex(
```

Comment:
Are you sure `rowIndex` is always unique? It's being used as a `key` here, which can lead to issues if not unique in this loop.

Reply:
Thanks @jonkoops I have added the `index` provided by the map(), and I'm using it as a fallback to ensure the uniqueness of the `key`.